### PR TITLE
fix: fixed text

### PR DESCRIPTION
**Pull request by acarranoqovery**

### CODE REVIEW

File: src/resources/variable/Variable.yaml [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
         type: string
         format: uuid
-      description: the id where the variable will be added
+      description: it filters the list by returning only the variables available to the selected parent_id (parent_id is the id of an application, cronjob or lifecycle job)
     - in: query
       name: scope
       schema:
         $ref: '../../schemas/enums/APIVariableScope.yaml'
-      description: the scope of the parent where the variable will be added
+      description: it filters the list by returning only the variables having the selected scope
```
Comment (pggb25) [inline src/resources/variable/Variable.yaml:16]: actually, it is the scope of the parent_id.

File: src/schemas/variable/VariableResponse.yaml [hunk 2]
```diff
@@ -15,10 +15,10 @@ allOf:
       service_id:
         type: string
         format: uuid
-        description: present only for `BUILT_IN` variable
+        description: The id of the service referenced by this variable. present only for `BUILT_IN` variable
       service_name:
         type: string
-        description: present only for `BUILT_IN` variable
+        description: The name of the service referenced by this variable. present only for `BUILT_IN` variable
```
[the remark this answers is not in the document]
Comment (acarranoqovery) [reply src/schemas/variable/VariableResponse.yaml:21]: in theory yes. only the core should be able to fill this field

File: src/resources/variable/Variable.yaml [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
+  description: Returns a list of variables. The result can be filtered by using the query parameters.
   operationId: listVariables
   parameters:
     - in: query
       name: parent_id
       schema:
         type: string
         format: uuid
-      description: the id where the variable will be added
+      description: it filters the list by returning only the variables available to the selected parent_id (parent_id is the id of an application, cronjob or lifecycle job)
```
[the remark this answers is not in the document]
Comment (acarranoqovery) [reply src/resources/variable/Variable.yaml]: fixed

File: src/resources/variable/Variable.yaml (hunk 1 above)
Comment (acarranoqovery) [reply src/resources/variable/Variable.yaml]: fixed

**Review by pggb25 — APPROVED** [review]

**Pull request merged**